app/InteractiveOptions/CreateDatabase.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+namespace App\InteractiveOptions;
+
+use App\Commands\NewCommand;
+use App\Support\BaseInteractiveOption;
+
+class CreateDatabase extends BaseInteractiveOption
+{
+    /**
+     * Option key.
+     *
+     * @var string
+     */
+    protected $key = 'database';
+
+    /**
+     * Performs interactively.
+     *
+     * @param $console
+     * @return BaseInteractiveOption
+     */
+    public function perform(NewCommand $console): BaseInteractiveOption
```

Comment:
OK, with these options, are they saving them as defaults? Or just for this create session? If they're saving them just for this seession I think I'd prefer they be a more traditional thing where you're prompted for each and just hit enter if you want it to do the default (I'm not sure, I could be talked out of it, but that's how it's feeling right now); walking through this whole UI feels really clumsy for a tool that's supposed to be all about speed. If they're setting the defaults, I think I'd rather "set my defaults" be a different action. I hope this question makes sense; I'm reading a *lot* here.